utils/html_components.py

```diff
@@ -81,3 +81,15 @@ def amp_price_conteiner(self, current_amp, new_amp, type_token_sell):
           </div>
         """
     )
+
```

Comment:
@yvesfracari we have to double check if everything is working on smaller screens. Here you component and the ones I've built previously are breaking. I'll add this as a todo on notion